java/java.mx.project/test/unit/src/org/netbeans/modules/java/mx/project/SuiteActionProviderTest.java

```diff
@@ -86,8 +86,9 @@ public void finished(boolean success) {
         MockProgress progress = new MockProgress();
 
         Lookup ctx = Lookups.fixed(fo, p, progress);
-        assertTrue("Clean is supported", ap.isActionEnabled(ActionProvider.COMMAND_CLEAN, ctx));
-        ap.invokeAction(ActionProvider.COMMAND_CLEAN, ctx);
+        assertTrue("Build is supported", ap.isActionEnabled(ActionProvider.COMMAND_BUILD, ctx));
+        // Do not run clean action here as it breaks next tests.
+        ap.invokeAction(ActionProvider.COMMAND_BUILD, ctx);
 
         assertTrue("Progress started", progress.started);
         progress.finished.await(10, TimeUnit.SECONDS);
```

Comment:
Since `build` may take more time than `clean`, consider to increase the timeout here.

Reply:
Done.